### PR TITLE
undo redo bi events

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/plugin-undo-redo/web/src/RedoButton.tsx [hunk 1]
```diff
@@ -46,6 +46,7 @@ const RedoButton: FunctionComponent<RedoButtonProps> = props => {
   const onClick = event => {
     event.stopPropagation();
     setEditorState(redo(getEditorState()));
+    helpers?.onPluginAddSuccess?.(UNDO_REDO_TYPE, 'redoButton', {});
```
Comment (danarish4) [inline packages/plugin-undo-redo/web/src/RedoButton.tsx:49]: This is the bi? click on undo\redo triggers onPluginAddSuccess? this is not make sense

File: packages/plugin-undo-redo/web/src/RedoButton.tsx [hunk 2]
```diff
@@ -46,6 +46,7 @@ const RedoButton: FunctionComponent<RedoButtonProps> = props => {
...
   };
```
Comment (danarish4) [inline packages/plugin-undo-redo/web/src/RedoButton.tsx:50]:
```suggestion
  const onClick = this.props.onClick
```

File: packages/plugin-undo-redo/web/src/createToolbar.tsx [hunk 3]
```diff
@@ -11,20 +11,24 @@ import type {
...
+  helpers,
 }: {
   t: TranslationFunction;
   getEditorState: GetEditorState;
   setEditorState: SetEditorState;
   settings: UndoRedoPluginEditorConfig;
   commonPubsub: Pubsub;
+  helpers: Helpers;
 }) => {
   return {
```
[the remark this answers is not in the document]
Comment (danarish4) [reply packages/plugin-undo-redo/web/src/createToolbar.tsx:33]: send this fuction both to external and components

File: packages/plugin-undo-redo/web/src/UndoButton.tsx [hunk 4]
```diff
@@ -46,6 +46,7 @@ const UndoButton: FunctionComponent<UndoButtonProps> = props => {
   const onClick = event => {
     event.stopPropagation();
     setEditorState(undo(getEditorState()));
+    helpers?.onPluginAddSuccess?.(UNDO_REDO_TYPE, 'undoButton', {});
   };
```
[the remark this answers is not in the document]
Comment (talevy17) [reply packages/plugin-undo-redo/web/src/UndoButton.tsx]: done

File: packages/plugin-undo-redo/web/src/createToolbar.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (talevy17) [reply packages/plugin-undo-redo/web/src/createToolbar.tsx:48]: done

**Pull request merged**